### DISCUSSION TITLE
the S orientations generated by Miller and Eulers are different in '1' specimenSymmetry

### DISCUSSION THREAD

**Discussion by Brian-Liu-rx** [mtex-toolbox/mtex#1776] [Ask Anything]
Hi, everyone，

   I generated two S orientations based on miller and eulers respectively and then calculated the misorientation between them.
   If orthorhombic specimenSymmetry is used, the misorientation is 0.5 degree, which seems good!
   However, when triclinic specimenSymmetry  is used, the misorientation is 38 degree, which makes me confused!
   Can anyone explain the reason behind?  Thanks a lot!

   Brian L

clear;
CS = crystalSymmetry.load('Al-Aluminum.cif');
SS = specimenSymmetry('1'); 
ori_S_bymiller = orientation.byMiller([1 2 3],[6 3 4],CS,SS); 
ori_S_byeuler = orientation.byEuler(59*degree,37*degree,63*degree,'ZXZ',CS,SS);
angle(ori_S_bymiller,ori_S_byeuler)./degree

**Comment by ralfHielscher** [comment]
Why do you think they should be equal?
